### ISSUE TITLE
RN 0.65: Error while dexing. The dependency contains Java 8 bytecode

### ISSUE THREAD

**Issue by andreialecu** [getsentry/sentry-react-native#1736]
The React Native 0.65 upgrade helper recommends removing these lines from `build.gradle`:

```
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
```

ref: https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.65.0

Once removed, sentry stops compiling with this error:

```
> Execution failed for DexingNoClasspathTransform: /Users/andreialecu/.gradle/caches/transforms-3/e47f952fd3d62178db1e85709ab1a9a9/transformed/jetified-sentry-5.1.0-beta.2.jar.
         > Error while dexing.
           The dependency contains Java 8 bytecode. Please enable desugaring by adding the following to build.gradle
           android {
               compileOptions {
                   sourceCompatibility 1.8
                   targetCompatibility 1.8
               }
           }
```

Opening this just so it can be tracked. Not sure what the actual solution would be.

**Comment by marandaneto** [comment]
done by https://github.com/getsentry/sentry-react-native/pull/1763/files a while ago

**Comment by AminDannak** [comment]
> done by https://github.com/getsentry/sentry-react-native/pull/1763/files a while ago

took me a while to find the exact version in which this change is available, but here it is.
It is available from version 3.0.3
